app/code/Magento/InventoryInStorePickup/Test/_files/create_in_store_pickup_quote_on_eu_website.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+use Magento\Quote\Api\CartManagementInterface;
+use Magento\Quote\Api\CartRepositoryInterface;
+use Magento\Quote\Api\Data\AddressInterface;
+use Magento\Quote\Api\Data\AddressInterfaceFactory;
+use Magento\Store\Api\StoreRepositoryInterface;
+use Magento\Store\Model\StoreManagerInterface;
+use Magento\TestFramework\Helper\Bootstrap;
+
+/** @var CartRepositoryInterface $cartRepository */
+$cartRepository = Bootstrap::getObjectManager()->get(CartRepositoryInterface::class);
+/** @var CartManagementInterface $cartManagement */
+$cartManagement = Bootstrap::getObjectManager()->get(CartManagementInterface::class);
+/** @var AddressInterfaceFactory $addressFactory */
+$addressFactory = Bootstrap::getObjectManager()->get(AddressInterfaceFactory::class);
+/** @var StoreRepositoryInterface $storeRepository */
+$storeRepository = Bootstrap::getObjectManager()->get(StoreRepositoryInterface::class);
+/** @var StoreManagerInterface\ $storeManager */
+$storeManager = Bootstrap::getObjectManager()->get(StoreManagerInterface::class);
+
+$cartId = $cartManagement->createEmptyCart();
+$cart = $cartRepository->get($cartId);
+$cart->setCustomerEmail('admin@example.com');
+$cart->setCustomerIsGuest(true);
+$store = $storeRepository->get('store_for_eu_website');
+$cart->setStoreId($store->getId());
+$storeManager->setCurrentStore($store->getCode());
+
+/** @var AddressInterface $address */
+$address = $addressFactory->create(
+    [
+        'data' => [
```

Comment:
I love how you save a US address to the Quote in fixture designed for EU Website :)